opmd_viewer/addons/pic/lpa_diagnostics.py

```diff
@@ -240,16 +240,17 @@ def get_divergence( self, t=None, iteration=None, species=None,
         ux, uy, uz, w = self.get_particle( var_list=['ux', 'uy', 'uz', 'w'],
                                            t=t, iteration=iteration,
                                            species=species, select=select )
-        # Calculate diveregence
+        # Calculate divergence
         div_x = w_std( np.arctan2(ux, uz), w )
         div_y = w_std( np.arctan2(uy, uz), w )
         # Return the result
         return( div_x, div_y )
 
-    def get_emittance( self, t=None, iteration=None, species=None,
-                       select=None ):
+    def get_emittance(self, t=None, iteration=None, species=None,
+                      select=None, kind='normalized', type='projected',
```

Comment:
Actually, I know realize that using `type` is a bad idea (since it's also a Python command). Sorry for suggesting this in the first place.

Could you maybe use `slicing` instead? (and replace throughout the function)